optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/jackson/LocalDateDeserializer.java

```diff
@@ -0,0 +1,35 @@
+package org.optaplanner.openshift.employeerostering.shared.jackson;
+
+import java.io.IOException;
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.core.JsonTokenId;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+public class LocalDateDeserializer extends JsonDeserializer<LocalDate> {
```

Comment:
Add a comment that clearly indicates when this class would become obsolete.

By copying them, we don't get automatic bugfixes as we upgrade to newer jackson versions, but that is indeed the lesser evil today. This is a very good, practical PR.
However, that situation won't stay the case: at some point every modern WildFly/EAP version will contain the jackson java.time support out of the box and we no longer need to duplicate them.
The comment should say something like "// TODO remove me when our minimum WildFly version is x and the minimum EAP version is y"